provider/datagen/Cargo.toml

```diff
@@ -106,6 +106,7 @@ use_wasm = ["icu_codepointtrie_builder/wasm"]
 # rule based segmenter data will not be generated.
 use_icu4c = ["icu_codepointtrie_builder/icu4c"]
 networking = ["dep:cached-path"]
+icu_provider_fs = ["dep:icu_provider_fs"]
```

Comment:
Thought: maybe the feature should be named `fs`? The feature isn't really about the dependency; it's about the functionality, and the dependency is a convenient side-effect.

Reply:
Yeah `icu_provider_fs` was a placeholer. I like `fs`, was also thinking `filesystem` to match `networking` but they're talking about different kinds of things

Reply:
I'd prefer `provider_fs` over `fs`. Datagen still makes extensive use of the filesystem.